src/ui/CategoryFacet/CategoryFacetSearch.ts

```diff
@@ -86,7 +87,9 @@ export class CategoryFacetSearch implements IFacetSearch {
     if (this.facetSearchElement.currentResult) {
       const currentResultPathData = this.facetSearchElement.currentResult.el.dataset.path;
       const delimiter = this.categoryFacet.options.delimitingCharacter;
-      this.categoryFacet.changeActivePath(currentResultPathData.split(delimiter));
+      const path = currentResultPathData.split(delimiter);
+      this.categoryFacet.logAnalyticsEvent(analyticsActionCauseList.categoryFacetSelect, path);
```

Comment:
Would it be true to say that every time we change the active path, we want to log an analytics event? or are there exceptions, like when clearing the breadcrumb as in the other PR?

I like the explicit call, just seeing if it makes sense to group them together.

Reply:
Yes, there are exceptions, so the explicit call seems simpler/straightforward.